### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#302] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152321621-c7bd7f09-241c-47da-b378-4826465eb9b8.png)

**Comment by danieldinella** [comment]
Essendo 10 e 15 già multipli di 5 analizziamo il caso 10 + 15. Questa situazione può essere riscritta come 2 * 5 + 3 * 5 = 5 * 5 che ovviamente é un multiplo di 5. Da ciò possiamo anche dedurre che il risultato sarà determinato dalla moltiplicazione tra (k * 2 + j * 3) * 5, dove k indica il numero di 10 e j il numero di 15, ed essendo sempre presente l'operatore 5 avremo sempre un multiplo di 5.

**Comment by Exyss** [comment]
![Esercizio 2](https://user-images.githubusercontent.com/65043207/152643451-601efcb6-5f75-4db5-8e25-62eef0e650a1.jpg)

**Comment by LuginiAndrea** [comment]
Per quanto le due soluzioni siano giuste, volevo aggiungerne una che probabilmente non è neanchè considerabile induzione.
Notiamo come 5 sia la minima differenza fra due numeri crescenti ottenuti dalla somma di $q \cdot 10 + 15 \cdot z, \forall q, z \in \mathbf{N}$, ottenuta sommando $15$ e togliendo $10$. Infatti le altre alternative sarebbero sommare $10$ e togliere $15$, il che ci farebbe però ottenere un numero decrescente (contro ipotesi), e sommare $10$ o $15$, dove si vede facilmente che la differenza sarebbe $>5$.

Il numero più piccolo come somma di 10 e 15 e, banalmente $10$.
Consideriamo ora la successione dei numeri crescenti con differenza minima dati dalla somma di $10$ e $15$, $x_n$, definita come $x_0 = 10, x_{n+1} = x_n + 5$.
$\forall n \geq 1$, possiamo vedere che $x_{n} = x_{n-1} + 5$. Procedendo ricorsivamente notiamo come la formula sia scrivibile come $$x_{n} = x_0 + \left(n\right) \cdot 5 = 10 + \left(n\right) \cdot 5 = 5 \cdot \left(2 + n \right)$$
Facilmente notabile come $\forall n \in \mathbf{N}, 5 \cdot \left(2 + n\right)$ sia divisibile per 5
